components/dashboard/sections/ActivityLog/ActivityDescription.tsx

```diff
@@ -38,7 +38,7 @@ export const getActivityVariables = (
   hasParent: Boolean(activity.account?.parent),
   Individual: () => (
     <LinkCollective
-      collective={activity.individual}
+      collective={activity.individual || activity.createdByAccount}
```

Comment:
I don't see any `createdByAccount` in https://github.com/opencollective/opencollective-api/blob/4cead49db818fba466b8f8f76ede3de0b0c0ef61/server/graphql/v2/object/Activity.ts